artifactory/commands/replication/create.go

```diff
@@ -68,7 +69,12 @@ func (rcc *ReplicationCreateCommand) Run() (err error) {
 		if key == "serverId" {
 			serverId = value.(string)
 		} else {
-			writersMap[key](&replicationConfigMap, key, value.(string))
+			if writerMapFunc, ok := writersMap[key]; ok {
```

Comment:
1. I think it will be more readable if we will check "!ok", and continue without the "else"
2. The variable name can be shorten just to "writer"